proxy/envoy/config.go

```diff
@@ -161,7 +161,12 @@ func build(instances []*model.ServiceInstance, services []*model.Service,
 	_ = buildFaultFilters(nil, nil)
 	_ = buildMixerCluster(mesh)
 
-	return listeners, clusters.Normalize()
+	clusters = clusters.Normalize()
+	for _, cluster := range clusters {
+		insertDestinationPolicy(config, cluster)
+	}
+
```

Comment:
Sometime in a future PR, we should decouple the cluster building from the listener building. 
buildListeners->buildFilters
buildClusters
composeConfig(listeners,clusters,tls_context)